### ISSUE TITLE
gRPC variable autocomplete – show time is terribly slow

### ISSUE THREAD

**Issue by deep-outcome** [getinsomnia/insomnia#3270]
**Describe the bug**
The variable autocomplete list is opening incredibly slow when using it in a gRPC request. Using it can result in broken layout.

**To Reproduce**
1. Create some environment and populate it with variables.
2. Create a gRPC request.
3. Try to use the autocomplete in the gRPC host section.
4. (Optional) Compare time-to-show to a REST request.

**Expected behavior**
List is shown in same time as for a REST.

**Screenshots**
Note the position of the autocomplete window.

![Bad gRPC autocomplete](https://user-images.githubusercontent.com/26486200/113987386-a0f1bc00-984e-11eb-9778-97c0f1d49a2b.png)

**Desktop (please complete the following information):**
Version: Insomnia 2021.2.2
Release date: 3/29/2021
OS: Windows_NT x64 10.0.19041
Electron: 9.3.5
Node: 12.14.1
V8: 8.3.110.13-electron.0
Architecture: x64
node-libcurl: libcurl/7.73.0-DEV OpenSSL/1.1.1g zlib/1.2.11 brotli/1.0.9 WinIDN libssh2/1.9.0_DEV nghttp2/1.41.0

**Additional context**
Compared to the REST autocomplete for the gRPC contains additional items not coming from an environment.

**Comment by deep-outcome** [comment]
Well, the bugfixing is not so popular as it was… 👎🏻 Fortunately anyone can use the new features instead, of course. 😕

**Comment by filfreire** [comment]
Hi @bravequickcleverfibreyarn, thank you for reporting this, and apologies for the radio silence for a long while.

I tried reproducing a bit your issue in the [latest Insomnia stable build](https://github.com/Kong/insomnia/releases/tag/core%402022.3.0), and was not able to.

As an experiment I tried pasting about 1000 environment variable entries, and tried using those within a gRPC request and didn't notice any major slowness.

![Screenshot 2022-05-23 at 11 04 13](https://user-images.githubusercontent.com/11976836/169796276-de78542c-522b-4dd3-9eba-7d92ccd07422.png)
![Screenshot 2022-05-23 at 11 04 08](https://user-images.githubusercontent.com/11976836/169796281-2f25b791-4eab-49bc-b57a-e9c66bfd63b5.png)


Are you still noticing slowness with our [latest build](https://github.com/Kong/insomnia/releases/tag/core%402022.3.0)?
